pkg/handler/internal/metrics/metrics.go

```diff
@@ -0,0 +1,35 @@
+// Copyright 2020 The Operator-SDK Authors
```

Comment:
Should we not cover it with the tests?

Reply:
https://github.com/operator-framework/operator-sdk/pull/3436#discussion_r455929337